### DISCUSSION TITLE
How do I use the CreatableSelect?

### DISCUSSION THREAD

**Discussion by MegaKeegMan** [JedWatson/react-select#5198] [Q&A]
I am not filing a bug yet, because maybe I am doing something wrong.

I add this to my component:
```
<CreatableSelect
  isMulti
  options={scopedReasons}
  onChange={handleChange}
/>
```
scopedReasons is an array of strings. onChange looks like this for now:
```
const handleChange = (newValue: OnChangeValue<string, true>) => {
  console.log(newValue);
};
```
Now I keep getting this error:
```
Error: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: object.
```
I have no idea why this is happening. Any ideas?
Here are my relevant imports:
```
import type {
  OnChangeValue,
} from "react-select";
import CreatableSelect from "react-select/creatable";
```

**Comment by Methuselah96** [comment]
You can't use string for options. Your options are to make them objects or use `getOptionValue` and `getOptionLabel`. See [this](https://react-select.com/upgrade-to-v2#simple-value) for more details.
